ui/components/multichain/select-action-modal-item/select-action-modal-item.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  FlexDirection,
+  IconColor,
+  TextAlign,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarIcon,
+  AvatarIconSize,
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../component-library';
+
+export const SelectActionModalItem = ({
+  actionIcon,
+  onClick,
+  showIcon,
+  primaryText,
+  secondaryText,
+  isDisabled,
+}) => {
+  const SelectActionItem = () => {
```

Comment:
I don't think we need this as a function.  At the top of the component, I think we can just do:

```
if (disabled) {
 return null;
}
```

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/20559/commits/42dab95363bb1926231a09ec4d2c2432c952d85c